### ISSUE TITLE
[Code Editor 2] Don't favour `/* */` comments on partial code selection!

### ISSUE THREAD

**Issue by thennothinghappened** [YoYoGames/GameMaker-Bugs#5305]
### Is your feature request related to a problem?

In the original code editor (and most other IDEs), using the keybind to comment out a block does one of two things:
1. For each line selected, prepends `// `.
2. _At the beginning of_ the first line, prepend `/*`, and _at the end of_ the last line append `*/`.

The original code editor does option 1. CE2 on the other hand does option 3, and places a block comment opening and closing exactly where the selection is.

Most times you intend to comment out a section of code, its as a quick measure to either debug or rewrite something. Having to exactly position your selection correctly whilst trying to do something _quickly_ isn't very efficient or fun!

### Describe the solution you'd like

I'd like CE2 to return to the original behaviour and prepend selected lines with `// `. Either option 1 or 2 is mostly acceptable, but I'd argue in favour of the former, as it shines better in a situation in which you then selectively uncomment part of the block but not all. Option 2 does not work well in this case - and this is all about making it as simple as possible for the developer.

In fact, CE2 actually already knows how to do this if you manage to select correctly, but seems to have the partial selection case intentionally setup to do a less helpful action.

### Describe alternatives you've considered

It could make sense to keep this behaviour when the user selection is on a _single_ line, in such cases where you want to comment a part of a line to replace it temporarily with something else. This doesn't make sense when the selection is over multiple lines though, which it currently does anyway.

**Comment by backYard321** [comment]
I agree with pretty much everything here, however i'd ALSO like keeping the `/* */` behaviour when multiple lines are selected by alt+dragging. EG:

![image](https://github.com/user-attachments/assets/081b7a47-af99-414e-8b1e-74651807ceed)

**Comment by backYard321** [comment]
also: 
![Image](https://github.com/user-attachments/assets/89427a90-72b9-42a6-aab4-bcad1436b37b)

**Comment by thennothinghappened** [comment]
Gonna go ahead and update this - CE2 now has a preference to _always_ use `//` comments, which is a _huge_ improvement over how it had been, but this doesn't cover the case where you may want to use `/* */` comments when editing single lines _only_, and `//` comments for multi-line selections (as discussed here.)
